disaster-recovery.md

```diff
@@ -0,0 +1,168 @@
+# Disaster Recovery
+
+This documentation describes how to recover from critical network failure scenarios.
+
+
+## Scenario 1: Dealing with compromised keys, or loss of network control
+
+The following section deals with the situation where different keys become compromised, or where the validators collectively lose control of the network, or critical components comprising the application stack.
+
+
+### Scenario 1.1: Ethereum key compromised
+
+In the event the Ethereum key is compromised, this key needs to be deactivated as soon as possible. As signatures issued with the Ethereum key are not time bound (i.e., a signed message can be used to authenticate a transaction weeks later), a key also should be considered compromised if the attacker had communication access to the HSM, as it is then unclear which and how many messages the attacker has signed (unless the HSM supports sufficient logging to analyse this case).
+
+The Ethereum key can be replaced by following the steps below.
+
+1. Notify the validators that control the `MultisigControl` contract that your Ethereum key has been compromised and needs to be updated.
+2. The `remove_signer` function must be executed on the `MultisigControl` contract to remove the compromised key. Each signatory on the `MultisigControl` contract needs to generate their own signature, and a single signature bundle will be submitted to the contract when calling `remove_signer`.
+3. After removing the compromised key, the new key for the validator must be added to the `MultisigControl` contract by calling the `add_signer` function. The signature bundle is generated in the same way as for (2).
+4. The Vega CLI can be used to generate signatures, please execute `vega bridge erc20 -h` for further instructions.
+
+
+### Scenario 1.2: Vega hot key compromised
+
+In the event that the Vega hot key is compromised, it can be replaced using the steps below.
+
+1. Terminate the affected node.
+2. Use the Vega master key to generate a new hot key (refer to [generating keys](https://github.com/vegaprotocol/networks#generating-vega-keys)).
+3. Submit a transaction to the Vega network, signed with your master key, authorising the new hot key. This is done by xxx.
```

Comment:
@JonRay15 "this is done by xxx" - should this be filled in or removed?

Reply:
Hmm... dont know on that part that was something Lewis had already done.  The same wording doesnt appear in the original doc, I think it can be removed.

Reply:
Have now removed this